src/main/java/org/dsa/iot/knx/DeviceFolder.java

```diff
@@ -160,12 +164,12 @@ void restoreLastSession() {
 
 		for (Node child : children.values()) {
 			Value restype = child.getAttribute(ATTR_RESTORE_TYPE);
-			if (restype != null && restype.getString().equals(ATTR_EDITABLE_FOLDER)) {
+			if (null != restype && restype.getString().equals(ATTR_EDITABLE_FOLDER)) {
```

Comment:
no I was talking about the `restype.getString().equals(ATTR_EDITABLE_FOLDER)` that should be `ATTR_EDITABLE_FOLDER.equals(restype.getString())` to avoid potential NPE